web/src/main/java/org/akaza/openclinica/controller/StudyController.java

```diff
@@ -657,9 +666,51 @@ private Study createSchemaStudy(HttpServletRequest request, Study study, UserAcc
         return schemaStudy;
     }
 
-    private void updateStudyConfigParameters(HttpServletRequest request, Study schemaStudy, StudyParameterConfig studyParameterConfig) {
+    private void updateStudyConfigParameters(HttpServletRequest request, Study schemaStudy, StudyParameterConfig studyParameterConfig , String templateID ,Boolean enrollmentCap) {
```

Comment:
I think this method should be generic such that it takes any study parameter and its key and adds an entry to the study parameter value table, if the entry doesn't exist.

Reply:
The way it is written currently, it doesn't scale well if we are adding new properties in the future.